_config.yml

```diff
@@ -6,7 +6,7 @@ author: Josh Davis
 description:
     List of sites with Two Factor Auth support which includes SMS, email,
     phone calls, hardware, and software.
-url: http://twofactorauth.org
+url: https://twofactorauth.org
```

Comment:
Should a comment be appended so that anyone else that stumbles upon this will know why HTTP - not HTTPS - is used?